rotkehlchen/tests/unit/test_blockchain_balances.py

```diff
@@ -146,9 +146,9 @@ def test_serialize(blockchain_balances):
 
     # change something and see it is also reflected in the serialized dict
     a.optimism[address2].assets[OPTIMISM_USDC_TOKEN] = Balance(amount=100, usd_value=100)
-    expected_serialized_dict['OPTIMISM'][address2]['assets'][OPTIMISM_USDC_TOKEN.serialize()] = {'amount': '100', 'usd_value': '100'}  # noqa: E501
+    expected_serialized_dict['optimism'][address2]['assets'][OPTIMISM_USDC_TOKEN.serialize()] = {'amount': '100', 'usd_value': '100'}  # noqa: E501
```

Comment:
here and elsewhere to not need to touch everything again each time we change serialization.

```suggestion
    expected_serialized_dict[chain.serialize()][address2]['assets'][OPTIMISM_USDC_TOKEN.serialize()] = {'amount': '100', 'usd_value': '100'}  # noqa: E501
```